src/FNA3D_Driver_D3D11.c

```diff
@@ -5137,6 +5390,12 @@ static FNA3D_Device* D3D11_CreateDevice(
 		? D3D_DRIVER_TYPE_WARP
 		: D3D_DRIVER_TYPE_UNKNOWN; /* Must be UNKNOWN if adapter is non-null according to spec */
 
+	/* Force debug mode if the user set the hint */
+	if (SDL_GetHintBoolean("FNA3D_D3D11_FORCE_DEBUG", SDL_FALSE))
+	{
+		debugMode = 1;
+	}
```

Comment:
We can put this in FNA3D.c, under `FNA3D_FORCE_DEBUG`

Reply:
Also addressed in latest commit.